client/src/utils/location.ts

```diff
@@ -1,39 +1,19 @@
-import { KaKaoRegion } from 'src/types';
+import api from 'src/api';
+import { location } from 'src/types';
```

Comment:
타입 맨 앞글자는 대문자로 컨벤션 맞추면 좋을 것 같습니다~
```suggestion
import { Location } from 'src/types';
```